skopt/optimizer/optimizer.py

```diff
@@ -467,7 +471,7 @@ def tell(self, x, y, fit=True):
                 # minimization starts from `n_restarts_optimizer` different
                 # points and the best minimum is used
                 elif self.acq_optimizer == "lbfgs":
-                    x0 = X[np.argsort(values)[:self.n_restarts_optimizer]]
+                    x0 = X[np.argsort(values)]
```

Comment:
Look at https://github.com/scikit-optimize/scikit-optimize/pull/284 for the benchmarks that lead to this change.